Glossary.md

```diff
@@ -1,6 +1,8 @@
 # Jollyballs Glossary
 
 ## General Terms
+* **Asseblage**- A term used when calling a game to indicate that the players should gather at the table.
```

Comment:
`playas` is preferred, but I don't feel strongly about this.

Reply:
A glossary item should probably include a pronunciation.

Reply:
/əˈseblɪdʒ/